base/dict.jl

```diff
@@ -147,38 +90,7 @@ end
 summary{T<:Union{KeyIterator,ValueIterator}}(iter::T) =
     string(T.name, " for a ", summary(iter.dict))
 
-function show(io::IO, iter::Union{KeyIterator,ValueIterator})
-    if !get(io, :multiline, false)
-        return show(io, collect(iter))
-    end
-    print(io, summary(iter))
-    isempty(iter) && return
-    print(io, ". ", isa(iter,KeyIterator) ? "Keys" : "Values", ":")
-    limit::Bool = get(io, :limit, false)
-    if limit
-        sz = displaysize(io)
-        rows, cols = sz[1] - 3, sz[2]
-        rows < 2 && (print(io, " …"); return)
-        cols < 4 && (cols = 4)
-        cols -= 2 # For prefix "  "
-        rows -= 2 # For summary and final ⋮ continuation lines
-    else
-        rows = cols = 0
-    end
-
-    for (i, v) in enumerate(iter)
-        print(io, "\n  ")
-        limit && i >= rows && (print(io, "⋮"); break)
-
-        if limit
-            str = sprint(0, show, v, env=io)
-            str = _truncate_at_width_or_chars(str, cols, "\r\n")
-            print(io, str)
-        else
-            show(io, v)
-        end
-    end
-end
+show(io::IO, iter::Union{KeyIterator,ValueIterator}) = show(io, collect(iter))
```

Comment:
this will be showing them as if they're a different type than they actually are? should at least have a header?